modules/dd-ix/monitoring.nix

```diff
@@ -0,0 +1,24 @@
+{pkgs, config, lib, ...}: 
+let
+  cfg = config.dd-ix.monitoring;
+in {
+  options.dd-ix.monitoring = {
+    enable = lib.mkEnableOption (lib.mdDoc "dd-ix monitoring");
+    #name = lib.mkOption {
+    #  type = lib.types.str;
+    #};
+  };
+
+  config = lib.mkIf cfg.enable {
+    services.prometheus.exporters = {
+      node = {
+	enable = true;
+	port = 9100;
+	listenAddress = (builtins.elemAt (builtins.split "/" config.dd-ix.microvm.v6Addr) 0); # removing cidr
```

Comment:
Please do **not** bind on explicit address (if not required for security or routing reasons). This will unnecessary break the services in case of a broken networking config (use `in6addr_any` if binding is required to enable ipv6 support).